### DISCUSSION TITLE
Any news on the relese of v1.0.7 ?

### DISCUSSION THREAD

**Discussion by ithinkido** [dbuezas/lgt8fx#189] [Q&A]
Does anyone have any info on when the new v1.0.7 library will be released with the EEPROM improvements mentioned  [here](https://github.com/dbuezas/lgt8fx/wiki/EEPROM-of-the-LGT8F328p-:-the-important-things-you-need-to-know)  ? I would be happy to test the new EEPROM functionality.

**Comment by dbuezas** [comment]
Hey @ithinkido sorry for the long sleep. Got distracted with another project.
Will try to test and make a release this weekend.

**Reply by ithinkido** [reply]
Thanks, I appreciate that.

**Reply by SuperUserNameMan** [reply]
wow, that is a looooooooooong weekend :-D

**Reply by dbuezas** [reply]
Darn I got into chips with wifi and got "slightly" distracted 😁
I promise I'll also write the steps so the other maintainers can also do it easily.

**Reply by SuperUserNameMan** [reply]
I've been slightly distracted away from LGT8 too, so I will not blame you !
My renewed interest in LGT8 is because I'm building an off-line archive of resources for all my "not yet used" MCU boards and modules. Internet is cool, but when it stops working, it's painful.
